code/_onclick/telekinesis.dm

```diff
@@ -79,14 +79,13 @@
 	START_PROCESSING(SSfastprocess, src)
 
 /obj/item/tk_grab/Destroy()
-	STOP_PROCESSING(SSfastprocess, src)
 	focus = null
-	tk_user = null
+	STOP_PROCESSING(SSfastprocess, src)
 	return ..()
 
 /obj/item/tk_grab/process()
 	if(check_if_focusable(focus)) //if somebody grabs your thing, no waiting for them to put it down and hitting them again.
-		update_appearance()
+		update_icon()
```

Comment:
```suggestion
		update_appearance()
```

Unintended revert? `update_icon()` is generally considered the deprecated proc compared to `update_appearance()`